core/src/main/java/com/yandex/yoctodb/util/immutable/impl/FixedLengthByteArraySortedSet.java

```diff
@@ -81,6 +81,42 @@ public Buffer get(final int i) {
         return elements.slice(((long) i) * elementSize, elementSize);
     }
 
+    @Override
+    public long getLongUnsafe(final int i) {
+        assert 0 <= i && i < size;
+
+        return elements.getLong(((long) i) * elementSize) ^ Long.MIN_VALUE;
```

Comment:
Here (and below) it might be faster to add `assert` that `elementSize == Long.BYTES` and replace `* elementSize` by bitwise shift.

Reply:
Nice idea. Took into account.